gradio/themes/__init__.py

```diff
@@ -0,0 +1 @@
+from .default import Default  # noqa: F401
```

Comment:
nit: `import gradio.themes as themes` in gradio/__init__.py so that VsCode and IDES can resolve the themes import.

Reply:
nice, thanks